replace DESCRIPTION with SUMMARY in mix_class

sschindler89 opened the pull request:
the "Description" field in the systemd service files is used to create log messages,
if we plainly copy the "DESCRIPTION" from our .bb recipes, the logs won't make much sense
the "SUMMARY" field usually contains a rather short description that works good with
systemd's expactation of a "Description"

if this is accepted we might have to adjust our "SUMMARY"s in the .bb recipes

thz commented:
meta-caros/classes/mix.bbclass replaces those @@-placeholders
SUMMARY is not replaced, so this needs to be adapted within this change.

see sed in https://github.com/carosio/meta-caros/blob/master/classes/mix.bbclass#L212

Commit:
replace DESCRIPTION with SUMMARY in mix_class

the "Description" field in the systemd service files is used to create log messages,
if we plainly copy the "DESCRIPTION" from our .bb recipes, the logs won't make much sense
the "SUMMARY" field usually contains a rather short description that works good with
systemd's expactation of a "Description"

sschindler89 commented:
updated according to @thz commentary
